src/diff-snapshot.js

```diff
@@ -154,16 +156,9 @@ function diffImageToSnapshot(options) {
       PNG.bitblt(
         receivedImage, compositeResultImage, 0, 0, imageWidth, imageHeight, imageWidth * 2, 0
       );
-
-      const input = { imagePath: diffOutputPath, image: compositeResultImage };
-      // image._packer property contains a circular reference since node9, causing JSON.stringify to
-      // fail. Might as well discard all the hidden properties.
-      const serializedInput = JSON.stringify(input, (name, val) => (name[0] === '_' ? undefined : val));
-
-      // writing diff in separate process to avoid perf issues associated with Math in Jest VM (https://github.com/facebook/jest/issues/5163)
-      const writeDiffProcess = childProcess.spawnSync('node', [`${__dirname}/write-result-diff-image.js`], { input: Buffer.from(serializedInput) });
-      // in case of error print to console
-      if (writeDiffProcess.stderr.toString()) { console.log(writeDiffProcess.stderr.toString()); } // eslint-disable-line no-console, max-len
+      // Set filter type to Paeth to avoid expensive auto scanline filter detection
+      const pngBuffer = PNG.sync.write(compositeResultImage, { filterType: 4 });
```

Comment:
you may want to add https://www.w3.org/TR/PNG-Filters.html to your comment for future engineers to reference.